### PR TITLE
Update README.md header

### PR DESCRIPTION
**Pull request by davidedistefano**
This PR updates the `README.md` layout visually, adding the full Solidus logo, improving the header and the Open Collective section.

**Description**
This PR could be the first of many improvements to this project `README.md`, like for example adding admin screenshots or a good project description. 

**Checklist:**
- [ ] I have followed [Pull Request guidelines](https://github.com/solidusio/solidus/blob/master/CONTRIBUTING.md#pull-request-guidelines)
- [ ] I have added a detailed description into each commit message
- [ ] I have updated Guides and README accordingly to this change (if needed)
- [ ] I have added tests to cover this change (if needed)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,84 @@
...
+  <a href="https://opencollective.com/solidus"><img src="https://opencollective.com/solidus/backers/badge.svg" alt="Backers on Open Collective"></a>
+  <a href="https://opencollective.com/solidus"><img src="https://opencollective.com/solidus/sponsors/badge.svg" alt="Sponsors on Open Collective"></a>
+  <br>
+  <a href="https://www.codetriage.com/solidusio/solidus"><img src="https://www.codetriage.com/solidusio/solidus/badges/users.svg" alt="Open Source Helpers"></a>
+  <a href="https://houndci.com"><img src="https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg" alt="Reviewed by Hound"></a>
+</p>
+
+<p align="center">
+  <strong>A free, open-source ecommerce platform that gives you complete control over your store.</strong>
+</p>
```
Comment (tvdeyen) [inline README.md]: I am not a fan of using more and more html in our readme file. First it is a markdown file and secondary this file is also meant to be read in an editor or terminal. Please do not see the GitHub preview as the single way to view this file.

**Comment by davidedistefano** [comment]
@tvdeyen @jacobherrington I understand your concern, but I think it's a matter of choice which kind of users to please when we talk about a README file, whether editor/terminal users or GitHub users. I know the first choice is more dev-friendly, but a lot of people use GitHub, and in my opinion, a good-looking README is good for the success of the platform. In fact, many successful projects, such as [Vue.js](https://github.com/vuejs/vue), have a hybrid (HTML/markdown) file. 

By the way, this is not a crucial change for now, and we can find a compromise, but I think, in the future, we should have more flexibility in terms of how to update the README, such as adding new content like images (which are not displayed in the editor/terminal) and sponsor logos.

**Comment by tvdeyen** [comment]
@davidedistefano the GitHub readme is not a marketing page. We have a nice looking site for this.

> a good-looking README is good for the success of the platform

A good looking readme is possible with plain markdown. Centering text is personal taste and does neither decide over the success of this plattform nor lead to a better looking readme.

**Comment by kennyadsl** [comment]
IMHO, the README is part of the marketing material, especially if the target of the thing to market is mostly made by developers. We can argue that some developers use the README in the editor and not via GitHub, but honestly, I only have my intuition here, no real data. 

Anyway, I think Thomas represents a part of users that prefer not to see HTML code in the README and we should listen to this feedback. 

I suppose that centering text is something that we can easily change, probably a nice-looking table with partners images is more concerning, right @davidedistefano?

**Comment by davidedistefano** [comment]
I removed the HTML tags where possible in favor of markdown.

File: README.md [hunk 2]
```diff
@@ -1,31 +1,54 @@
...
-<a href="https://supergood.software/"><img src="https://images.opencollective.com/proxy/images?src=https%3A%2F%2Fopencollective-production.s3-us-west-1.amazonaws.com%2F3bbb1440-727f-11e9-a366-37673cc38cee.png&height=100"></a>
-<a href="https://karmacreative.io/"><img src="https://images.opencollective.com/proxy/images?src=https%3A%2F%2Fopencollective-production.s3-us-west-1.amazonaws.com%2Fab94d2a0-7253-11e9-a366-37673cc38cee.png&height=100"></a>
+|<a href="https://www.enginecommerce.com/" target="_blank"><img src="https://images.opencollective.com/proxy/images?src=https%3A%2F%2Flogo.clearbit.com%2Fenginecommerce.com&height=100"></a>|<a href="https://supergood.software/" target="_blank"><img src="https://images.opencollective.com/proxy/images?src=https%3A%2F%2Fopencollective-production.s3-us-west-1.amazonaws.com%2F3bbb1440-727f-11e9-a366-37673cc38cee.png&height=100"></a>|<a href="https://karmacreative.io/" target="_blank"><img src="https://images.opencollective.com/proxy/images?src=https%3A%2F%2Fopencollective-production.s3-us-west-1.amazonaws.com%2Fab94d2a0-7253-11e9-a366-37673cc38cee.png&height=100"></a>|
```
Comment (tvdeyen) [inline README.md]: These should also be writable as markdown. HTML should not be necessary

**Comment by kennyadsl** [comment]
@tvdeyen we can reduce HTML further in favor or Markdown but some images need to be resized (they are SVG) and it cannot be done with GitHub markdown, unfortunately. Do you have any suggestion?

**Comment by tvdeyen** [comment]
@kennyadsl the Nebulab logo could have a `viewport` in the svg with the correct size and there will not need a size attibute on the `img` tag. See http://tutorials.jenkov.com/svg/svg-viewport-view-box.html

**Comment by davidedistefano** [comment]
Now everything is written in Markdown except for the Solidus logo, which was already written in HTML.

**Pull request merged**